.NET/Microsoft.Recognizers.Text.Options/Extractors/IChoiceExtractorConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Microsoft.Recognizers.Text.Options.Extractors
+{
+    public interface IChoiceExtractorConfiguration
```

Comment:
I see that in some places structure classes are named "Choice" and in others "Options". Let's rename all for consistency once the discussion on the best name converges to one.

Reply:
Renamed Options recognizer to Choice recognizer

Reply:
👍